src/rcj_flask.py

```diff
@@ -94,3 +94,72 @@ def get_runs():
 def get_runs_competition(competition):
     return jsonify({'runs': g.rcj.get_runs_competition(competition)})
 
+def api_v2_check_auth(request):
+    username = request['referee']['name']
+    password = request['referee']['auth']
+    return g.rcj.check_referee_password(username, password)
+
+@app.route('/crud', methods=['GET'])
+def crud():
+    return send_from_directory('../public', 'crud.html')
+@app.route('/crud/main.js', methods=['GET'])
+def crud_main_js():
+    return send_from_directory('../public', 'main.js')
+
+@app.route('/api/v2/sql', methods=['POST'])
+def sql():
+    if not request.is_json:
+        return jsonify({'error': 'not json'}), 400
+    try:
+        req = request.json
+    except ValueError as e:
+        return jsonify({'error': str(e)}), 400
```

Comment:
We already checked if it is json. No need for the try-except clause here.
I think we can also delete the try-except clause above in the "/api/v1/submit_run"-function.